pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -142,6 +142,7 @@
 , docbook_xsl
 , docbook_xml_dtd_42
 , docbook_xml_dtd_45
+, withLogTrace ? false
```

Comment:
why not make it true?

Reply:
This causes `LOG_TRACE` to be defined, which then definess `log_trace*` functions actually doing logging.

I assume some of these are quite heavy perf-wise, so they're not compiled in by default.